### ISSUE TITLE
Ctrl + scroll event isn't cancellable

### ISSUE THREAD

**Issue by olmokramer** [qutebrowser/qutebrowser#3836]
At least Chromium and Firefox (I've also tested on Falkon, but it has the same behavior as qutebrowser) allow javascript on the page to disable zooming with ctrl+scroll. This is useful because some websites use ctrl+scroll to zoom in/out on maps (e.g. https://ah.nl/winkels).

#### Reproducer

Save the HTML below as /tmp/test.html (or whatever), then `:open /tmp/test.html` and scroll the page while holding the control key, the page content is zoomed in or out.

```html
<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8">
	<title>WheelEvent ctrl test</title>
</head>
<body>
	<script>
	document.addEventListener('wheel', function onWheel(event) {
          	event.preventDefault();
	}, true);
	</script>
</body>
</html>
```

#### Version

```
$ qutebrowser --version
qutebrowser v1.2.1
Git commit: 69d642cab (2018-04-09 19:11:10 +1200)
Backend: QtWebEngine (Chromium 61.0.3163.140)

CPython: 3.6.4
Qt: 5.10.1
PyQt: 5.10.1

sip: 4.19.8
colorama: no
pypeg2: 2.15
jinja2: 2.10
pygments: 2.2.0
yaml: 3.12
cssutils: no
attr: 17.4.0
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebKitWidgets: no
pdf.js: no
sqlite: 3.23.0
QtNetwork SSL: OpenSSL 1.1.0h  27 Mar 2018

Style: QFusionStyle
Platform: Linux-4.15.14-1-ARCH-x86_64-with-arch, 64bit
Linux distribution: Arch Linux (arch)
Frozen: False
Imported from /usr/lib/python3.6/site-packages/qutebrowser
Qt library executable path: /usr/lib/qt/libexec, data path: /usr/share/qt

Paths:
cache: /home/olmo/.cache/qutebrowser
config: /home/olmo/.config/qutebrowser
data: /home/olmo/.local/share/qutebrowser
runtime: /run/user/1000/qutebrowser
system data: /usr/share/qutebrowser

Uptime: 0:00:00
```

**Comment by The-Compiler** [comment]
Not sure whether this is fixable at all, because we get and handle the scroll event from Qt before the backend (and thus also JavaScript) even sees it.

**Comment by Fierthraix** [comment]
Ran into the same issue, one would expect that it works in passthrough mode, but it doesn't currently.

**Comment by lucc** [comment]
I hit the same issue. Additional info: the map also zooms (in addition to qutebrowser)

First thing I tried was passthrough mode (with ctrl-v) but that did not change the behaviour which is surprising. I was expecting qutebrowser to stop any processing of key and special mouse events (other than clicking on links. Notably "right click" is passed through in passthrough mode as can be seen on http://xkcd.com/1975).

Maybe it is a reasonable feature request in the scope of this issue to pass through the "ctrl-scroll" event if passthrough mode is active?

**Comment by lucc** [comment]
PS: I just now read Fierthraix's post thoroughly, sorry for the duplication :confused:

**Comment by The-Compiler** [comment]
Yeah - not sure if we can make it cancellable, but I agree it should be passed through in passthrough mode.
